Pages/navbar.html

```diff
@@ -60,7 +51,7 @@
                 >
                     Shop
                 </a>
-            </li>
+            <!-- </li>
             <li class="relative group">
```

Comment:
Did you mean to comment out the next list item instead of the previous closing tag?